nethttp/server.go

```diff
@@ -75,6 +75,10 @@ func MWSpanObserver(f func(span opentracing.Span, r *http.Request)) MWOption {
 //		}),
 //   )
 func Middleware(tr opentracing.Tracer, h http.Handler, options ...MWOption) http.Handler {
+	return MiddlewareFunc(tr, h.ServeHTTP, options...)
+}
+
+func MiddlewareFunc(tr opentracing.Tracer, h func(http.ResponseWriter, *http.Request), options ...MWOption) http.Handler {
```

Comment:
please add docs

Reply:
I've added documentation to the function. Tried to refer readers to the godoc on `Middleware(...)` so as to reduce duplication.